content/docs/operating/security.md

```diff
@@ -34,10 +36,10 @@ that service discovery system.
 
 Scraped targets may be run by untrusted users. It should not by default be
 possible for a target to expose data that impersonates a different target.  The
-`honor_labels` option removes this protection, as can certain relabelling
-setups.
+`honor_labels` option removes and certain relabelling setups remove this
```

Comment:
This change doesn't parse.

Reply:
Your phrasing is overly convoluted, I tried to untangle it.

Reply:
I am fine with different wording, of course.

Reply:
Your wording still doesn't parse.